front/components/assistant/AssistantDetailsButtonBar.tsx

```diff
@@ -94,6 +154,15 @@ export function AssistantDetailsButtonBar({
               }}
               icon={BracesIcon}
             />
+            <DropdownMenuItem
+              label={isExporting ? "Exporting..." : "Export to YAML"}
+              onClick={(e) => {
+                e.stopPropagation();
+                void handleExportToYAML();
+              }}
+              icon={DocumentIcon}
+              disabled={isExporting}
+            />
```

Comment:
`nit`: we can mimic what we have on the name and description suggestion when it's loading
```typescript
<Button
  icon={isGenerating ? () => <Spinner size="xs" /> : SparklesIcon}
  variant="outline"
  size="xs"
...
```